static_report/src/components/shared/Tables/TableList/CRTableListItem/index.tsx

```diff
@@ -65,15 +58,16 @@ export function CRTableListItem({
           <Flex gap={2}>
             {children}
             {isExpanded && (
-              <ChakraLink
-                as={Link}
-                to={`/tables/${name}`}
+              <Flex
+                as="a"
                 data-cy="cr-navigate-report-detail"
+                onClick={(event) => {
+                  event.stopPropagation();
```

Comment:
This can stop even propagation, and prevent trigger `Accordion` expand behavior when navigating to the next page.